core/primitives-core/src/profile.rs

```diff
@@ -157,146 +202,32 @@ impl fmt::Debug for ProfileData {
-            Cost::ExtCost { ext_cost_kind: ExtCosts::storage_remove_key_byte } => 44,
-            Cost::ExtCost { ext_cost_kind: ExtCosts::storage_remove_ret_value_byte } => 45,
-            Cost::ExtCost { ext_cost_kind: ExtCosts::storage_has_key_base } => 46,
-            Cost::ExtCost { ext_cost_kind: ExtCosts::storage_has_key_byte } => 47,
-            Cost::ExtCost { ext_cost_kind: ExtCosts::storage_iter_create_prefix_base } => 48,
-            Cost::ExtCost { ext_cost_kind: ExtCosts::storage_iter_create_prefix_byte } => 49,
-            Cost::ExtCost { ext_cost_kind: ExtCosts::storage_iter_create_range_base } => 50,
-            Cost::ExtCost { ext_cost_kind: ExtCosts::storage_iter_create_from_byte } => 51,
-            Cost::ExtCost { ext_cost_kind: ExtCosts::storage_iter_create_to_byte } => 52,
-            Cost::ExtCost { ext_cost_kind: ExtCosts::storage_iter_next_base } => 53,
-            Cost::ExtCost { ext_cost_kind: ExtCosts::storage_iter_next_key_byte } => 54,
-            Cost::ExtCost { ext_cost_kind: ExtCosts::storage_iter_next_value_byte } => 55,
-            Cost::ExtCost { ext_cost_kind: ExtCosts::touching_trie_node } => 56,
-            Cost::ExtCost { ext_cost_kind: ExtCosts::promise_and_base } => 57,
-            Cost::ExtCost { ext_cost_kind: ExtCosts::promise_and_per_promise } => 58,
-            Cost::ExtCost { ext_cost_kind: ExtCosts::promise_return } => 59,
-            Cost::ExtCost { ext_cost_kind: ExtCosts::validator_stake_base } => 60,
-            Cost::ExtCost { ext_cost_kind: ExtCosts::validator_total_stake_base } => 61,
-            Cost::WasmInstruction => 62,
-            Cost::ExtCost { ext_cost_kind: ExtCosts::read_cached_trie_node } => 63,
-            Cost::ExtCost { ext_cost_kind: ExtCosts::alt_bn128_g1_multiexp_base } => 64,
-            Cost::ExtCost { ext_cost_kind: ExtCosts::alt_bn128_g1_multiexp_element } => 65,
-            Cost::ExtCost { ext_cost_kind: ExtCosts::alt_bn128_pairing_check_base } => 66,
-            Cost::ExtCost { ext_cost_kind: ExtCosts::alt_bn128_pairing_check_element } => 67,
-            Cost::ExtCost { ext_cost_kind: ExtCosts::alt_bn128_g1_sum_base } => 68,
-            Cost::ExtCost { ext_cost_kind: ExtCosts::alt_bn128_g1_sum_element } => 69,
-            #[cfg(feature = "protocol_feature_ed25519_verify")]
-            Cost::ExtCost { ext_cost_kind: ExtCosts::ed25519_verify_base } => 70,
-            #[cfg(feature = "protocol_feature_ed25519_verify")]
-            Cost::ExtCost { ext_cost_kind: ExtCosts::ed25519_verify_byte } => 71,
-        }
-    }
-}
-
-impl Index<Cost> for ProfileData {
-    type Output = u64;
-
-    fn index(&self, index: Cost) -> &Self::Output {
-        &self.data[index.profile_index()]
-    }
-}
-
-impl IndexMut<Cost> for ProfileData {
-    fn index_mut(&mut self, index: Cost) -> &mut Self::Output {
-        &mut self.data[index.profile_index()]
-    }
-}
+mod profile_v2;
```

Comment:
The conventional style and the one that we have been following across nearcore, I believe, is to have `mod` definitions at the top of the file, after the `use` items.